apps.toml

```diff
@@ -1354,6 +1369,7 @@ category = "system_tools"
 level = 7
 state = "working"
 url = "https://github.com/Yunohost-Apps/httpsh_ynh"
+antifeatures = ["deprecated-software"]
```

Comment:
(one commit last month but otherwise inactive, idk if we put the flag or not, it's still more modern than shellinabox?)

Reply:
Wooopsies, shouldn't have flagged this one ! It's not deprecated at all :
* the core "smart" code is actually `ttyd` and it's maintained
* httpsh is basically a 6 line bash script that calls `login` and ssh.

I could even copy this bash script inside the app package and change the manifest upstream url to ttyd… dunno

Reply:
```suggestion
url = "https://github.com/Yunohost-Apps/httpsh_ynh"
```